Buildings/Resources/Documentation/userGuide/source/development.rst

```diff
@@ -249,22 +642,46 @@ The following rules need to be followed, in addition to the guidelines described
    Make sure sequences are tested for all modes of operation, and as applicable, for winter, shoulder
    and summer days.
 
+#. Perform code text and, if applicable, figure inspection to:
+      - ensure there are no remaining typos
+      - ensure no element `public` was added automatically after the `protected` code section
 
+#. Use full paths to blocks, that is paths that start with `Buildings.Controls...`
 
-.. _sec_val:
+#. Final steps for a pull requests:
```

Comment:
Remove. This belongs to how to make a pull request. It is not specific to control sequences.